### PR TITLE
[Intl/Normalizer] Add Unicode conformance tests

### PR DESCRIPTION
**Pull request by nicolas-grekas**

### CODE REVIEW

File: tests/Intl/Normalizer/NormalizerTest.php [hunk 1]
```diff
@@ -78,4 +78,66 @@ public function testNormalize()
...
+                $this->assertSame($c[3], pn::normalize($c[1], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[2], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[3], pn::NFKC));
+                $this->assertSame($c[3], pn::normalize($c[4], pn::NFKC));
+
+                $this->assertSame($c[4], pn::normalize($c[0], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[1], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[2], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[3], pn::NFKD));
+                $this->assertSame($c[4], pn::normalize($c[4], pn::NFKD));
```
Comment (stof) [inline tests/Intl/Normalizer/NormalizerTest.php]: should we really call the stub directly here ? Shouldn't it call the polyfilled API instead, relying on the custom test listener ?

Comment (nicolas-grekas) [inline tests/Intl/Normalizer/NormalizerTest.php]: I'd say that testing the native implantation again the unicode spec is out of scope. We have a standardize reference here so no need for the native php reference. At least that's the reasoning I've had for now.

Comment (nicolas-grekas) [inline tests/Intl/Normalizer/NormalizerTest.php]: OK, done

**Comment by nicolas-grekas** [comment]
So, now that unicode conformance tests run against the native implementations, we have to find a common subset that works for all the various ICU versions into the wild.

**Comment by nicolas-grekas** [comment]
Green

File: tests/Intl/Normalizer/NormalizerTest.php [hunk 2]
```diff
@@ -44,14 +44,14 @@ public function testIsNormalized()
         $c = 'déjà';
         $d = in::normalize($c, pn::NFD);
 
-        $this->assertTrue(pn::isNormalized(''));
-        $this->assertTrue(pn::isNormalized('abc'));
-        $this->assertTrue(pn::isNormalized($c));
-        $this->assertTrue(pn::isNormalized($c, pn::NFC));
+        $this->assertTrue(!!normalizer_is_normalized(''));
```
Comment (stof) [inline tests/Intl/Normalizer/NormalizerTest.php]: why ``!!`` ? is it returning something else than a boolean ?

Comment (nicolas-grekas) [inline tests/Intl/Normalizer/NormalizerTest.php]: Yes on hhvm, I reported it btw

Comment (stof) [inline tests/Intl/Normalizer/NormalizerTest.php]: I suggest adding a comment referencing the HHVM issue, to remember why this casting is there.

And I think ``(bool)`` would be more readable for PHP devs (the ``!!`` is common in JS to save bytes, but you are the only PHP dev I know who try to make the PHP code look like minified code)

**Pull request merged**